.ci_support/linux_64_.yaml

```diff
@@ -1,24 +1,18 @@
 c_compiler:
 - gcc
 c_compiler_version:
-- '9'
+- '13'
 cdt_name:
-- cos6
+- conda
 channel_sources:
-- conda-forge,defaults
+- conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-comp7
+- quay.io/condaforge/linux-anvil-x86_64:alma9
 lua:
 - '5'
-pin_run_as_build:
-  zlib:
-    max_pin: x.x
 target_platform:
 - linux-64
-zip_keys:
-- - cdt_name
-  - docker_image
 zlib:
-- '1.2'
+- '1'
```

Comment:
not sure why this didn't get unpinned before. so it is pinning zlib to 1.2 and now allowing co-installation with zlib 1.3